test/caesar_test.rb

```diff
@@ -1,2 +1,27 @@
-# YOUR TESTS GOES HERE
-# I PITY THE FOOL WHO DOESNT WRITE TESTS
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/caesar'
+require 'pry'
+
+class CaesarTest < Minitest::Test
+
+  def test_caeser_exists
+    caesar = Caesar.new("The quick", 3)
```

Comment:
I see what you're trying to do here, but remember that the spec doesn't say that you're passing in the rotation and the plaintext when you initialize, it only gets it when you give it to the translation method.